assets/src/edit-story/components/inspector/document/publish/index.js

```diff
@@ -115,6 +121,7 @@ function PublishPanel() {
 
   const generateCanvas = useCallback(async () => {
     try {
+      const html2canvas = (await import('html2canvas')).default;
       const canvas = await html2canvas(fullbleedContainer);
```

Comment:
The canvas size can be different depending on screen size.

We need to ensure the resulting screenshot is always the same size though (640px x 853px as per the [guidelines](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)))

This 3:4 aspect ratio is important. So I don't think we can use this container as-is.

Any ideas how we can best achieve this?

Reply:
No, because it just cannot render any part of the selection frame at all.